src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/SHA256.cs

```diff
@@ -36,21 +30,22 @@ public Implementation()
                 HashSizeValue = _hashProvider.HashSizeInBytes * 8;
             }
 
-            protected sealed override void HashCore(byte[] array, int ibStart, int cbSize)
-            {
+            protected sealed override void HashCore(byte[] array, int ibStart, int cbSize) =>
                 _hashProvider.AppendHashData(array, ibStart, cbSize);
-            }
 
-            protected sealed override byte[] HashFinal()
-            {
-                return _hashProvider.FinalizeHashAndReset();
-            }
+            protected override void HashCore(ReadOnlySpan<byte> source) =>
```

Comment:
In the name of making forward progress here, I'll update this with the style of the particular project, and will open an issue to track removing the sealed modifiers from across corefx.

> maintained this discipline for far longer than corefx has been around

That doesn't matter.  Lots of projects have come together with lots of different styles into corefx, and we're trying to maintain a consistent style across the whole repo.